### ISSUE TITLE
checkout.php bug with hard coded country and enhancements

### ISSUE THREAD

**Issue by jbruggeling** [strangerstudios/paid-memberships-pro#521]
checkout.php line 448
`<input type="hidden" name="bcountry" value="US" />`

Should the country not get its value from $pmpro_default_country to allow use outside of the US

When using long / international address there is no code to use the states.php include for a dropdown box. The below code would benefit for being attached to the long address too. 

`if($state_dropdowns === true || $state_dropdowns == "names")
							{
								global $pmpro_states;
							?>
							<select name="bstate" class=" <?php echo pmpro_getClassForField("bstate");?>">
								<option value="">--</option>
								<?php
									foreach($pmpro_states as $ab => $st)
									{
								?>
									<option value="<?php echo esc_attr($ab);?>" <?php if($ab == $bstate) { ?>selected="selected"<?php } ?>><?php echo $st;?></option>
								<?php } ?>
							</select>
							<?php
							}
							elseif($state_dropdowns == "abbreviations")
							{
								global $pmpro_states_abbreviations;
							?>
								<select name="bstate" class=" <?php echo pmpro_getClassForField("bstate");?>">
									<option value="">--</option>
									<?php
										foreach($pmpro_states_abbreviations as $ab)
										{
									?>
										<option value="<?php echo esc_attr($ab);?>" <?php if($ab == $bstate) { ?>selected="selected"<?php } ?>><?php echo $ab;?></option>
									<?php } ?>
								</select>
							<?php
							}
							else
							{
							?>
							<input id="bstate" name="bstate" type="text" class="input <?php echo pmpro_getClassForField("bstate");?>" size="2" value="<?php echo esc_attr($bstate)?>" />
							<?php
							}`

**Comment by strangerstudios** [comment]
I agree that the US value here should be set to the default country.

I'm not sure what you are requesting RE the state dropdowns. Long addresses are used by default. Our preferred way now of doing states via dropdowns is through this add on: https://github.com/strangerstudios/pmpro-state-dropdowns

**Comment by jbruggeling** [comment]
Regarding the states drop down it will not become a drop down with the below filters in place.

add_filter("pmpro_longform_address", '__return_true');
add_filter("pmpro_state_dropdowns", '__return_true');

The code section from if($state_dropdowns === true || $state_dropdowns == "names") would need to be placed under the long address section in checkout,php too to allow the above filters to work under both address formats

**Comment by jbruggeling** [comment]
Any update on getting this fixed?
